src/Shared/Common/ParticipantScreeningHelper.cs

```diff
@@ -0,0 +1,62 @@
+using System.Globalization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace NHS.ServiceInsights.Common;
+
+public class PaginationHelper
+{
+    private readonly ILogger _logger;
+    public PaginationHelper (ILogger logger)
+    {
+        _logger = logger;
+    }
+
+    public (bool IsValid, int page, int pageSize, DateTime startDate, DateTime endDate) ValidateQuery(HttpRequestData req)
```

Comment:
For this method can we follow the 'Try' pattern as seen with TryParse

e.g. Name it something like TryValidatePaginationQuery and return a bool indicating success/failure and output the results using the `out` parameters